perch/organisation.py

```diff
@@ -387,25 +417,32 @@ def update(self, user, **kwargs):
                 yield OAuthSecret.create(user, client_id=self.id)
 
     @coroutine
+    def can_approve(self, user, **data):
+        """
+        Only sys admins can approve a service
+        :param user: a User
+        :param data: data that the user wants to update
+        """
+        is_external = data.get('service_type', self.service_type) == 'external'
+        raise Return(user.is_admin() or is_external)
```

Comment:
I originally thought so too, but the behave test specifically check that only Hub admins can update services. Thinking back on it I remember a discussion where we decided to limit this as an extra level of validation, so that before a service can be fully registered with the hub it has to go through a manual approval step with a Hub Admin.